posts/2021/04/spot-the-differences.md

```diff
@@ -0,0 +1,213 @@
+
+**[Juanita]** Since we ended up with more than 30 issues, we had to start a 
+“triaging phase.” We had to label, triage, organize, and prioritize issues 
+according to “urgency” and importance. This issue tracker became our main 
+tool to keep up with all the plans for the future!
+
+**[Isabela]** Juanita did wonderful work tracking our progress through issues 
+and keeping us all accountable, but we were still left with a long list of 
+issues to triage—long enough that it wasn’t all getting in Spyder 5. To have 
+the greatest impact on Spyder, we started with the issues that had 
+influence on Spyder as a whole. Toolbars, icons, and colors are something y
+ou will always encounter from the first impression to the most recent, so it 
+made sense to start thinking about those big picture issues first.
+
+### 4. Digging deeper into the dark hole
+
+**[Isabela]** When prioritizing the audit feedback for Spyder 5, each pass 
+seemed to get to a deeper layer of the problem. For example, what started as 
+issues to make [tooltips more legible](https://github.com/spyder-ide/ux-improvements/issues/2) and improve the variable explorer’s 
+[color coding](https://github.com/spyder-ide/ux-improvements/issues/7) soon became the realization that we weren’t sure exactly what 
+blue was already being used for much of Spyder’s interface. It got more 
+complicated when we found out how many colors were hard coded across multiple 
+files or defined by an external project. Eventually, the problem changed from 
+the color contrast of tool tips to an [unsustainable approach for managing 
+color](https://github.com/spyder-ide/ux-improvements/issues/13) across the two default Spyder themes rooted in a non-Spyder repo. Work 
+at each step did build up into a larger solution, but it’s worth noting that 
+it isn’t what we set out to do in the first place. 
+
+### 5. What witchcraft does Isabela do in the background?
+
+**[Juanita]** One of the most important parts of the process was designing 
+the mock ups for the new ideas that we came up with for the interface which  
+is definitely not our expertise. So... how did the designs magically appear 
+on our Github issues?
+
+**[Isabela]** First things first, it isn’t actually witchcraft even if it 
+looks magical from the outside. How I work depends somewhat on what problem 
+e are trying to solve, so let’s use the design of [custom icons](https://github.com/spyder-ide/ux-improvements/issues/33#issuecomment-776376943) for Spyder 5 
+as an example. Once I had a defined list of icons to work on, I needed to 
+spend time making progress on my own. Research on best practices for the 
+relevant area of design is where I started; in this case, I knew we were 
+going to be working with [Material Design Icons’](https://materialdesignicons.com/) specifications. After that, 
+I did a combination of pen-and-paper sketching and working digitally based on 
+the existing icons in Spyder and Material Design Icons while I kept note of 
```

Comment:
I @isabela-pf wrote it like this because it is important to make clear that the distribution is called "Material Design Icons" from https://materialdesignicons.com and not "Material Design" from https://material.io/design.